ci/run-tests.py

```diff
@@ -33,7 +33,7 @@
 
 MAX_TRIES = 5
 
-dfhack = 'Dwarf Fortress.exe' if sys.platform == 'win32' else './dfhack'
+dfhack = 'Dwarf Fortress.exe' if sys.platform == 'win32' else './dwarfort'
```

Comment:
We should keep this as `./dfhack` on Linux for now so that we can see output.

Reply:
done